roboflow/__init__.py

```diff
@@ -67,14 +69,117 @@ def auth(api_key):
     return Roboflow(api_key, w)
 
 
+def login(workspace=None, force=False):
+    conf_location = os.getenv(
+        "ROBOFLOW_CONFIG_DIR",
+        default=os.getenv("HOME") + "/.config/roboflow/config.json",
+    )
+
+    if os.path.isfile(conf_location) and not force:
+        write_line(
+            "You are already logged into Roboflow. To make a different login, run roboflow.login(force=True)."
+        )
+        return
+
+    if workspace is None:
+        write_line(
+            "visit " + APP_URL + "/auth-cli" " to get your authentication token."
+        )
+    else:
+        write_line(
+            "visit "
+            + APP_URL
+            + "/auth-cli/?workspace="
+            + workspace
```

Comment:
Brad TODO: hook into this in core repo

Reply:
@yeldarby what this? I just authed via new account in incognito and it seemed pretty good

Reply:
If you have multiple workspaces, selecting the one you need

Reply:
Ok - I think it's OK for now, new user flow will be main concern for this method